src/desktop/components/search_bar/collections/search.coffee

```diff
@@ -23,21 +26,34 @@ module.exports = class Search
     "?#{@data()}"
 
   data: ->
-    data = visible_to_public: true, fair_id: @fairId, size: @size
+    data = visible_to_public: true, fair_id: @fairId, size: @size, agg: @agg
     "#{qs.stringify data}&term=%QUERY" # Requires an unencoded percent character
 
-  parseResults: (items) ->
-    if @restrictType?
-      @restrictType = [@restrictType] unless _.isArray @restrictType
-      _.filter items, (item) =>
-        _.contains @restrictType, item?.owner_type
-    else
-      items
-
-  parse: (items, query) ->
-    # HACK filter out sensitive results
-    items = _.reject items, (item) ->
+  filterResults: (items) ->
+    _.reject items, (item) ->
       JSON.stringify(item).match(/kippenberger|zoe.*leonard/i)
-    @results.reset _.map @parseResults(items), (item) =>
+
+  parseResults: (items, aggregations) ->
+    if aggregations
+      results = []
+      Object.keys(items).forEach((key) ->
+        if items[key].length > 0
+          firstItem = items[key].shift()
+          firstItem.displayHeading = true
```

Comment:
The first item in each category gets these additional properties, so the template knows to render the category 'heading'.

Reply:
:+1: